templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -1099,6 +1099,15 @@ export default {
           errMsgs.push("Please select a question")
         }
       }
+
+      if (
+        this.node.mediaDuration &&
+        this.node.mediaType !== "video" &&
+        this.node.mediaType !== "h5p"
+      ) {
+        this.node.mediaDuration = undefined
+      }
+
```

Comment:
For clarity, would it be possible to separate this from the `validateNode` method? Since `validateNode` focuses on checking the node inputs, maybe the code to clear the duration could be put in `handleSubmit`, for example.